functionary/functionary/urls.py

```diff
@@ -40,6 +40,7 @@
         name="redoc",
     ),
     path("ui/", include("ui.urls")),
+    path("accounts/", include("allauth.urls")),
```

Comment:
This feels like it belongs in the ui apps urls .  Is there a reason it has to be here?